functions/ts/src/kubeval.ts

```diff
@@ -0,0 +1,173 @@
+import {
```

Comment:
Note: this removes the Git history showing this was contributed by David. Maybe we should rethink extracting out functions into this repo. Instead we can have a demo-functions and real-functions in the SDK repo. WDYT? This would also simplify how many repos we have to touch.

Reply:
I'd rather keep real functions in this repo, it seems easier to modularize this way and we can include not just typescript functions but any others like shell, starlark, etc.